apps/backoffice/src/pages/services/[serviceId]/edit-service.tsx

```diff
@@ -51,6 +56,10 @@ export default function EditService() {
           title: t("notifications.validationError"),
         }),
       );
+      logToMixpanel("IO_BO_SERVICE_CREATE_END", {
```

Comment:
Should be `IO_BO_SERVICE_EDIT_END`, correct?

Reply:
[fixed](https://github.com/pagopa/io-services-cms/pull/1018/commits/f19b230da57c7cb7ec1e72c997f7c35dec8f4ae9)